### ISSUE TITLE
Suggestions for a PR

### ISSUE THREAD

**Issue by VasasA** [backdrop-contrib/og#35]
@laryn Please see my suggestions below. If you find some useful suggestions, I’ll be happy to create a PR to them. 
1. tests/og_test.module: [Replace](https://docs.backdropcms.org/change-records/formnode_edit_form-removed): `empty($form['#node_edit_form'])` > `$form_state['build_info']['base_form_id'] != 'node_form'`
2. og_ui.test: [Replace](https://docs.backdropcms.org/change-records/formatusername-changed-userformatname) `format_username()` > `user_format_name()`
3. og_ui.module: Implement `og_ui_config_info()`
4. og_access.module Implement `og_access_config_info()`
5. og_access: Create `og_access.settings.json`
6. og_access.install: Implement `og_access_update_1000()`
7. og_access.install: Implement `og_access_update_last_removed()`
8. og_context: Replace `dynamic variable in file ...` with real default value: `og_context_update_1000()`, `og_context.settings.json`
9. og_context: Remove unused `og_context_negotiation_type` setting: `og_context_update_1000()`, `og_context.settings.json`
10. og_context.module: Improve `label` in `og_context_config_info()`: 'Module name settings' > 'Organic groups context settings'
11. og.module: Improve `label` in `og_config_info()`: 'Module name settings' > 'Organic groups settings'
12. Replace `variable_get()` with `config_get()`: og_access.module, og.module, og_ui.admin.inc
13. Replace `variable_set()` with `config_set()`: og.test, og_ui.test
14. og.install: `og_update_1000()`: Remove lines of settings with `dynamic variable in file ...` default value.
15. og.install: `og_uninstall()`: Remove `variable_del()` and `db_delete('variable')`.
16. og_context.install: `og_context_uninstall()`: Remove `variable_del()` (Empty function.)
17. og_access.install `og_access_uninstall()`: Remove `variable_del()` (Empty function.)
18. og_ui.install `og_ui_uninstall()`: Remove `variable_del()` (Empty function.)
19. og_access.install: Remove some functions: `og_access_schema()`, `og_access_update_7000()`, `og_access_update_7200()`
20. og.test: Remove unnecessary classes: `OgMigrate7000TestCase`, `OgMigrate7200TestCase`
21. og_ui.test: Remove unnecessary class: `OgUiMigrate7000TestCase`
22. Replace 'drupal' with 'backdrop': *.test files and og.api.php
23. Remove unnecessary `getInfo()` from *.test files.
24. Improve README.md: Upgrading Organic Groups from Drupal 7 ([source](https://github.com/backdrop-contrib/rules/wiki/Upgrading-Rules-from-Drupal-7))

**Comment by laryn** [comment]
Yes please 👍

**Comment by VasasA** [comment]
@laryn  I created the PR. My PR modifies the `og.test` file, @docwilmot's PR modifies and moves this file into the `tests` subdirectory. I hope it does not generate conflicts.

**Comment by VasasA** [comment]
@laryn If you merge my PR and Docwilmot's PR, this generates 23 conflicts. 22 in `og.test` and one in `og.tests.info`. Sorry.
I can help resolve the conflicts manually.

**Comment by VasasA** [comment]
@laryn I merged the two PR in my local test environment, and I resolved the conflicts manually. You can download the fixed files:
[og_test_files.zip](https://github.com/VasasA/converting/blob/master/og_test_files.zip)

**Comment by laryn** [comment]
Sounds great -- can you include those changes into your PR so it's easier to squash/merge here?

**Comment by VasasA** [comment]
I tried it, but it didn't reduce the number of conflicts. The fixed files can be useful only after the merging. This will result the best possible result. (I merged my PR first, Docwilmot's PR second.)

**Comment by VasasA** [comment]
@laryn I found a possible solution to avoid the conflicts: If Docwilmot and I include the fixed files into both PRs, then the merging will not generate conflicts. Should I consult Docwilmot?

**Comment by laryn** [comment]
@VasasA Perhaps you can force push on top of your current PR? Or start a new PR with all the relevant changes included if that is cleaner?

**Comment by VasasA** [comment]
> Perhaps you can force push on top of your current PR? Or start a new PR with all the relevant changes included if that is cleaner?

I think these solutions cannot prevent and reduce the conflicts when you merge both PRs. I think there are two possible solutions:
- You merge the PRs, and resolve the conflicts by `og_test_files.zip`.
- Docwilmot and I include and commit the fixed files into both PRs (no need to force), and the merging will not generate conflicts.

**Comment by laryn** [comment]
Closing this one with all current PRs merged. New PRs can file new issues. Thanks @VasasA !
